Client/SymfonyClient.php

```diff
@@ -65,10 +65,12 @@ class SymfonyClient implements ClientInterface
      */
     public function __construct(
         SessionInterface $session,
-        EnvironmentBuilderInterface $environmentBuilder = null
+        EnvironmentBuilderInterface $environmentBuilder = null,
+        \AppKernel $kernel
```

Comment:
Variables with default value should be last parameters indeed, so $kernel should be optional as well